lib/middleware/errors.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars*/
```

Comment:
Is this here because of the next argument? Could use `/*eslint no-unused-vars: ["error", { "varsIgnorePattern": "next" }]*/` as it's more specific and sort of documents why that comment is there.

Reply:
_sweet_